cache/cache.c

```diff
@@ -0,0 +1,101 @@
+#include "cache.h"
+#include "./config.h"
+
+/* private data used by cache_t */
```

Comment:
How about renaming it to `backend-redis.c`, which denotes the cache backend facilitated by Redis?

Reply:
Yeah, I think this is great. We can have different targets in bazel BUILD files in this way.